frontend/src/components/maps/PopupView.tsx

```diff
@@ -43,5 +71,5 @@ export const PopupView: React.FC<IPopupViewProps> = ({
   if (propertyTypeId === PropertyTypes.DRAFT_BUILDING) {
     return <p>This is a draft marker for building: {propertyDetail?.name ?? 'New Building'}</p>;
   }
-  return null;
+  return <></>;
```

Comment:
Ah, this makes more sense

Reply:
as far as I'm aware returning null in a render method is valid jsx, but I don't mind this syntax either.